soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11118,6 +11118,18 @@ void Player_Update(Actor* thisx, GlobalContext* globalCtx) {
         this->actor.scale.z = 0.001f;
     }
 
+    if (paperLink == 1) {
```

Comment:
This can probably be combined here:
```suggestion

    if (paperLink != 0) {
         this->actor.scale.x = paperLink == 1 ? 0.001f : 0.01f;
         this->actor.scale.y = 0.01f;
         this->actor.scale.z =  paperLink == 2 ? 0.001f : 0.01f;
     }
```

Reply:
I should’ve done this for gravity